releasenotes/notes/dump-terra-0_24-47c5d4fb92f1d97b.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The minimal required Qiskit-Terra version was bumped to 0.24. This version enables
+    new pulse amplitude representation of (``amp``, ``angle``) pair, 
+    improved user-communication for deprecated functionalities, 
+    and fast marginalization of the composite experiment result data.
```

Comment:
Minor point, but technically we already support fast marginalization without 0.24.

Reply:
Yes, let's remove this because I'll add a release note about it in 0.5.2.

Reply:
Okey done. Strictly speaking new pulse repr is the feature of 0.23. But having two notes for terra version update (0.22->0.23, 0.23->0.24) in the release note of 0.6 looks bit weird.

Reply:
Agreed, and the current statement is not incorrect. 0.24 enables new pulse repr because it contains 0.23 which enables it ;)